src/apps/backend/modules/password_reset_token/internal/store/password_reset_token_model.py

```diff
@@ -3,11 +3,13 @@
 from typing import Annotated, Any, Optional
 from pydantic import BaseModel, Field, validator, ConfigDict
 
-def object_id_validate(v: ObjectId | str) -> ObjectId | str:
-  assert ObjectId.is_valid(v), f'{v} is not a valid ObjectId'
-  if isinstance(v, str):
-    return ObjectId(v)
-  return str(v)
+def object_id_validate(v: ObjectId | str) -> ObjectId:
```

Comment:
Looks like we can move this out and add it in a different module (say, `objectid` ?) instead? We can reuse it then.

Reply:
Yeah, added this in `utils` under `object_id` module.